### ISSUE TITLE
Sentry print output to console upon terminate

### ISSUE THREAD

**Issue by shayownbackup** [getsentry/raven-python#1307]
When sentry python is terminating 'raven-python/raven/transport/threaded.py', method 'main_thread_terminated'.

Sentry is printing to output.
This cause damage to our process who relies on system output.

how can I solve this ?

thank you

**Comment by untitaker** [comment]
Thanks for taking the time to do this :) 

FYI, the new SDK does not have this issue.

**Comment by shayownbackup** [comment]
I can still see it on the master branch here 
[https://github.com/getsentry/raven-python/blob/master/raven/transport/threaded.py](url)

**Comment by untitaker** [comment]
Yes.The new sdk is at https://github.com/getsentry/sentry-python, where you filed the original issue
